SwiftWisdom/Rx/Rx+Extensions.swift

```diff
@@ -23,17 +23,15 @@ infix operator <-> : Binding
 infix operator <- : Binding
 infix operator >>> : Disposing
 
-public func <- <T>(property: AnyObserver<T>, variable: Observable<T>) -> Disposable {
-    return variable
-        .observeOn(MainScheduler.instance)
-        .bindTo(property)
+public func <- <T: ObserverType, O: ObservableType>(property: T, variable: O) -> Disposable where T.E == O.E {
+    return variable.observeOn(MainScheduler.instance).bindTo(property)
 }
 
-public func <- <T>(property: AnyObserver<T>, variable: Variable<T>) -> Disposable {
+public func <- <T: ObserverType, O>(property: T, variable: Variable<O>) -> Disposable where T.E == O {
```

Comment:
There is actually only one test for composite disposable here.

The tests wouldn't need to be updated because the functions are made more generic.

I will add some tests for these.